RoosterPlanner.UI/src/app/pages/admin/admin.component.ts

```diff
@@ -13,33 +16,53 @@ import{CreateProjectComponent} from "../../components/create-project/create-proj
 })
 export class AdminComponent implements OnInit {
   projects: Project[] = [];
-  listOfProjects:Project[][]=[]
-  tempList:Project[]=[];
+  listOfProjects: Project[][] = []
+  listOfAdmins: User[][] = []
+  tempListProjects: Project[] = [];
+  tempListAdmins: User[] = [];
   loaded: boolean = false;
 
-  administrators = [
-    {name: "Corné"},
-    {name: "JW"},
-    {name: "Yannick"},
-    {name: "Joanne"},
-  ]
+  administrators: User[] = []
 
-  constructor(public dialog:MatDialog,private projectService: ProjectService, private router: Router) {
+  constructor(public dialog: MatDialog, private projectService: ProjectService, private router: Router, private userService: UserService) {
   }
 
   async ngOnInit(): Promise<void> {
     await this.projectService.getProject().then(x => {
       this.projects = x;
+      this.splitProjects();
+    });
 
-      for(let i=0; i<this.projects.length; i++){
-        this.tempList.push(this.projects[i])
-        if(this.tempList.length===5||i===this.projects.length-1){
-          this.listOfProjects.push(this.tempList);
-          this.tempList=[];
-        }
-      }
+    await this.userService.getAdministrators().then(x => {
+      this.administrators = x;
+      this.spitAdministrators();
       this.loaded = true;
+
     });
+
+  }
+
+  spitAdministrators() {
+    for (let i = 0; i < this.administrators.length; i++) {
+      this.tempListAdmins.push(this.administrators[i])
+      if (this.tempListAdmins.length === 5 || i === this.administrators.length - 1) {
```

Comment:
Magic number (5), wat is dit?

Reply:
5 items per list. Het is nu expliciet aangegeven